examples/platform_channel/lib/main.dart

```diff
@@ -47,7 +47,7 @@ class _PlatformChannelState extends State<PlatformChannel> {
     });
   }
 
-  void _onError(PlatformException error) {
+  FutureOr _onError(Object error) {
```

Comment:
That's weird though - according to stream.dart:
```
   * The [onError] callback must be of type `void onError(error)` or
   * `void onError(error, StackTrace stackTrace)`. If [onError] accepts
   * two arguments it is called with the error object and the stack trace
   * (which could be `null` if the stream itself received an error without
   * stack trace).
...
  StreamSubscription<T> listen(void onData(T event),
      {Function onError, void onDone(), bool cancelOnError});
```

Also analyzer won't be happy with this:
```
[hint] This function declares a return type of 'FutureOr', but doesn't end with a return statement. (examples/platform_channel/lib/main.dart, line 50, col 3)
[lint] Specify type annotations. (examples/platform_channel/lib/main.dart, line 50, col 3)
```

Reply:
maybe void _onError(Object error) is better

Reply:
Yes, that would be better.